sdk/eventhub/azure-eventhub/azure/eventhub/_consumer_client.py

```diff
@@ -86,6 +87,20 @@ class EventHubConsumerClient(ClientBase):
     :paramtype checkpoint_store: ~azure.eventhub.CheckpointStore
     :keyword float load_balancing_interval: When load-balancing kicks in. This is the interval, in seconds,
      between two load-balancing evaluations. Default is 10 seconds.
+    :keyword float partition_ownership_expiration_interval: A partition ownership will expire after this number
+     of seconds. Every load-balancing evaluation will automatically extend the ownership expiration time.
+     Default is 6 * load_balancing_interval.
+    :keyword str or ~azure.eventhub.LoadBalancingStrategy load_balancing_strategy: When load-balancing kicks in,
```

Comment:
I'm not sure whether it's good to allow user to pass raw "str" as Enum, it's not deterministic and user may get typo.

Allowing just Enum seems to be the correct way to me, or do you have any thoughts on the raw "str" input?

Reply:
We should support strings directly. In fact, our enum models in the generated clients inherit from string for this very reason :)

Reply:
@annatisch , is this a general guideline?

We have another Enum "transport_type" which accepts "Amqp" and "AmqpOverWebsockets", should we support str as well?

And same question applies to service bus receive mode "PeekLock" and "ReceiveAndDelete"